src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -87,19 +88,26 @@ export class AccessibleBufferWidget extends DisposableStore {
 			cursorBlinking: this._configurationService.getValue('terminal.integrated.cursorBlinking'),
 			readOnly: true
 		};
-		this._accessibleBuffer = document.createElement('div');
-		this._accessibleBuffer.setAttribute('role', 'document');
-		this._accessibleBuffer.ariaRoleDescription = localize('terminal.integrated.accessibleBuffer', 'Terminal buffer');
-		this._accessibleBuffer.classList.add('accessible-buffer');
-		this._editorContainer = document.createElement('div');
-		this._bufferEditor = this.add(this._instantiationService.createInstance(CodeEditorWidget, this._editorContainer, editorOptions, codeEditorWidgetOptions));
+		this._editorWidget = this.add(this._instantiationService.createInstance(CodeEditorWidget, this._editorContainer, editorOptions, codeEditorWidgetOptions));
+		this._accessibleBuffer.replaceChildren(this._editorContainer);
+		this._xtermElement.insertAdjacentElement('beforebegin', this._accessibleBuffer);
+
 		this._focusTracker = this.add(dom.trackFocus(this._editorContainer));
+		this._focusedContextKey = TerminalContextKeys.accessibleBufferFocus.bindTo(this._contextKeyService);
+		this._trackFocus();
+
+		this._initializeTerminalListeners();
+		this._initializeEditorListeners();
+		this._updateEditor();
+	}
+
+	private _trackFocus(): void {
 		this.add(this._focusTracker.onDidFocus(() => this._focusedContextKey.set(true)));
 		this.add(this._focusTracker.onDidBlur(() => this._focusedContextKey.reset()));
-		this._accessibleBuffer.replaceChildren(this._editorContainer);
-		this._xtermElement.insertAdjacentElement('beforebegin', this._accessibleBuffer);
-		this.add(Event.runAndSubscribe(this._xterm.raw.onResize, () => this._bufferEditor.layout({ width: this._xtermElement.clientWidth, height: this._xtermElement.clientHeight })));
-		this._bufferEditor.onKeyDown((e) => {
+	}
```

Comment:
nit: Instead of making it a member function that's called a single time, spacing it out and using a comment is a nice way of doing it without adding the (tiny) overhead of an extra function and extra lines to define/call it:

```ts
		// Setup focus tracker
		this._focusTracker = this.add(dom.trackFocus(this._editorContainer));
		this._focusedContextKey = TerminalContextKeys.accessibleBufferFocus.bindTo(this._contextKeyService);
		this.add(this._focusTracker.onDidFocus(() => this._focusedContextKey.set(true)));
		this.add(this._focusTracker.onDidBlur(() => this._focusedContextKey.reset()));

		// Setup terminal listeners
		...

		// Setup editor listeners
		...
```